src/pages/payment-confirmation.js

```diff
@@ -64,14 +64,15 @@ class PaymentConfirmation extends React.Component {
                 <Col md={5} mdOffset={1}>
                   <H2>What happens now?</H2>
                   <P>
-                    You should shortly recieve an order confirmation and receipt
-                    and further details about the training you’ve signed up to.
-                    Just check your inbox soon (be sure to check your spam
-                    folder if you can’t see it).
+                    You should shortly receive an order confirmation and receipt
+                    in your email. Your ticket along with further details about
+                    the training you’ve signed up to will be sent within 24
+                    hours. Just check your inbox soon (be sure to check your
+                    spam folder if you can’t see it).
```

Comment:
Lena asked for this change in text as it was not clear to some people and she has received a few questions on this